### DISCUSSION TITLE
What kind of language server to use?

### DISCUSSION THREAD

**Discussion by andreichalapco** [STEllAR-GROUP/hpx#6297] [Q&A]
Hi,

I'm sadly pretty new with the hpx library and I wanted to get feedback on what language server to use in order to get the most of the valuable error messages when using the library.

I'm interested about this, because when using "clangd", the language server gives me a lot of errors and the use is not that good.
Is there a preferred language server in the community?

**Comment by hkaiser** [comment]
For me, Visual Studio works reasonably well.
